### ISSUE TITLE
titles sometimes fail to appear

### ISSUE THREAD

**Issue by grahamperrin** [captnfab/FireTitle#13]
This issue may be: 

- more likely to occur where a session has many tabs; and/or
- an extension conflict. 

Shortly before Firefox 55 was released: with 54.0.1 I typically worked with more than four hundred tabs, and the issue was occasional. 

Now with 55.0.1 I have a significantly greater number of tabs – 521, 18 of which are loaded – and the issue is frequent. 

## Example

![2017-08-15 07 34 09](https://user-images.githubusercontent.com/192271/29304894-425d86e4-8185-11e7-8a4e-debd95db542a.png)

Normally I don't have windows so small (the tiling above was solely for this bug report). 

An overview of normally sized windows: 

![2017-08-15 07 36 34](https://user-images.githubusercontent.com/192271/29304898-47a94458-8185-11e7-8bf7-e2caccef6ee8.png)

## Workarounds

It's usually enough to simply: 

1. select another tab in the same window
2. re-select the previously affected tab. 

## Raw data from `about:support`

```json
{
  "application": {
    "name": "Firefox",
    "osVersion": "FreeBSD 12.0-CURRENT",
    "version": "55.0.1",
    "buildID": "20170810060847",
    "userAgent": "Mozilla/5.0 (X11; FreeBSD amd64; rv:55.0) Gecko/20100101 Firefox/55.0",
    "safeMode": false,
    "updateChannel": "release",
    "supportURL": "https://support.mozilla.org/1/firefox/55.0.1/FreeBSD/en-GB/",
    "numTotalWindows": 11,
    "numRemoteWindows": 0,
    "remoteAutoStart": false,
    "currentContentProcesses": 0,
    "maxContentProcesses": 7,
    "autoStartStatus": 7,
    "keyGoogleFound": true,
    "keyMozillaFound": false
  },
  "modifiedPreferences": {
    "accessibility.typeaheadfind.flashBar": 0,
    "browser.cache.disk.filesystem_reported": 1,
    "browser.cache.disk.hashstats_reported": 1,
    "browser.cache.disk.smart_size.first_run": false,
    "browser.cache.frecency_experiment": 3,
    "browser.cache.disk.smart_size.use_old_max": false,
    "browser.cache.disk.capacity": 358400,
    "browser.download.useDownloadDir": false,
    "browser.fixup.alternate.enabled": false,
    "browser.places.smartBookmarksVersion": 8,
    "browser.search.openintab": true,
    "browser.search.suggest.enabled": false,
    "browser.sessionstore.upgradeBackup.latestBuildID": "20170810060847",
    "browser.sessionstore.postdata": -1,
    "browser.sessionstore.interval": 60000,
    "browser.sessionstore.restore_pinned_tabs_on_demand": true,
    "browser.startup.homepage_override.mstone": "55.0.1",
    "browser.startup.homepage": "about:profiles",
    "browser.startup.homepage_override.buildID": "20170810060847",
    "browser.tabs.tabMinWidth": 175,
    "browser.tabs.warnOnClose": false,
    "browser.tabs.remote.force-enable": false,
    "browser.tabs.remote.autostart.2": true,
    "browser.tabs.remote.autostart": true,
    "browser.tabs.tabMaxWidth": 300,
    "browser.tabs.animate": false,
    "browser.tabs.remote.force-disable": false,
    "browser.urlbar.lastSuggestionsPromptDate": 20170807,
    "browser.urlbar.doubleClickSelectsAll": false,
    "browser.urlbar.formatting.enabled": false,
    "browser.urlbar.maxRichResults": 12,
    "browser.urlbar.daysBeforeHidingSuggestionsPrompt": 3,
    "browser.urlbar.default.behavior": 0,
    "browser.urlbar.searchSuggestionsChoice": true,
    "browser.urlbar.userMadeSearchSuggestionsChoice": true,
    "dom.push.userAgentID": "b0ac2d48900a4b78a3e8b89e72e0149a",
    "dom.max_script_run_time": 0,
    "dom.max_chrome_script_run_time": 0,
    "dom.ipc.processCount": 7,
    "extensions.lastAppVersion": "55.0.1",
    "font.name.serif.x-western": "serif",
    "font.default.x-western": "sans-serif",
    "font.name.monospace.x-western": "monospace",
    "font.internaluseonly.changed": false,
    "font.name.sans-serif.x-western": "sans-serif",
    "general.useragent.locale": "en-GB",
    "gfx.use_text_smoothing_setting": true,
    "media.webrtc.debug.log_file": "/tmp/WebRTC.log",
    "media.peerconnection.ice.default_address_only": true,
    "media.gmp-manager.lastCheck": 1502772641,
    "media.gmp.storage.version.observed": 1,
    "media.peerconnection.ice.no_host": true,
    "media.gmp-manager.buildID": "20170810060847",
    "network.http.proxy.pipelining": true,
    "network.prefetch-next": false,
    "network.dns.disablePrefetch": true,
    "network.http.speculative-parallel-limit": 0,
    "network.negotiate-auth.trusted-uris": "https://staff.brighton.ac.uk/",
    "network.http.pipelining": true,
    "network.http.pipelining.max-optimistic-requests": 4,
    "network.http.pipelining.aggressive": true,
    "network.http.max-persistent-connections-per-proxy": 256,
    "network.cookie.cookieBehavior": 3,
    "network.http.pipelining.ssl": true,
    "network.http.pipelining.abtest": false,
    "network.http.pipelining.maxrequests": 32,
    "network.http.pipelining.reschedule-timeout": 15000,
    "network.http.pipelining.reschedule-on-timeout": true,
    "network.automatic-ntlm-auth.trusted-uris": "https://staff.brighton.ac.uk/",
    "network.predictor.cleaned-up": true,
    "network.negotiate-auth.delegation-uris": "https://staff.brighton.ac.uk/",
    "network.http.pipelining.maxsize": 300000,
    "network.http.max-connections": 256,
    "network.cookie.prefsMigrated": true,
    "network.http.pipelining.read-timeout": 60000,
    "places.history.expiration.transient_current_max_pages": 125328,
    "places.database.lastMaintenance": 1502299047,
    "plugin.disable_full_page_plugin_for_types": "application/pdf",
    "plugins.ctprollout.cohortSample": "0.132635",
    "plugins.ctprollout.cohort": "excluded",
    "plugins.click_to_play": false,
    "print.print_paper_width": "  8.27",
    "print.print_to_file": false,
    "print.print_margin_top": "0.5",
    "print.print_paper_name": "iso_a4",
    "print.print_orientation": 0,
    "print.print_duplex": 0,
    "print.print_margin_right": "0.5",
    "print.print_paper_height": " 11.69",
    "print.print_unwriteable_margin_right": 25,
    "print.print_unwriteable_margin_left": 25,
    "print.print_paper_size_unit": 0,
    "print.print_unwriteable_margin_bottom": 56,
    "print.print_paper_data": 0,
    "print.print_oddpages": true,
    "print.print_bgcolor": false,
    "print.print_bgimages": false,
    "print.print_evenpages": true,
    "print.print_margin_left": "0.5",
    "print.print_shrink_to_fit": true,
    "print.print_unwriteable_margin_top": 25,
    "print.print_margin_bottom": "0.5",
    "print.print_in_color": true,
    "print.print_page_delay": 50,
    "print.print_resolution": 1200,
    "privacy.clearOnShutdown.history": false,
    "privacy.history.custom": true,
    "privacy.donottrackheader.enabled": true,
    "privacy.clearOnShutdown.formdata": false,
    "privacy.usercontext.about_newtab_segregation.enabled": true,
    "privacy.clearOnShutdown.passwords": false,
    "privacy.userContext.enabled": true,
    "privacy.donottrackheader.value": 1,
    "privacy.clearOnShutdown.sessions": false,
    "privacy.trackingprotection.introCount": 20,
    "privacy.clearOnShutdown.cookies": false,
    "privacy.userContext.longPressBehavior": 2,
    "privacy.clearOnShutdown.downloads": false,
    "security.OCSP.enabled": 0,
    "security.OCSP.require": true,
    "services.sync.declinedEngines": "passwords,bookmarks",
    "services.sync.lastPing": 1502695310,
    "services.sync.lastSync": "Tue Aug 15 2017 07:46:38 GMT+0100 (BST)",
    "services.sync.engine.adblockplus": true,
    "services.sync.engine.greasemonkey": true,
    "services.sync.engine.bookmarks": false,
    "services.sync.engine.prefs.modified": false,
    "services.sync.engine.passwords": false,
    "storage.vacuum.last.index": 1,
    "storage.vacuum.last.places.sqlite": 1502679707
  },
  "lockedPreferences": {},
  "javaScript": {
    "incrementalGCEnabled": true
  },
  "accessibility": {
    "isActive": false,
    "forceDisabled": 0
  },
  "libraryVersions": {
    "NSPR": {
      "minVersion": "4.16",
      "version": "4.16"
    },
    "NSS": {
      "minVersion": "3.32",
      "version": "3.32"
    },
    "NSSUTIL": {
      "minVersion": "3.32",
      "version": "3.32"
    },
    "NSSSSL": {
      "minVersion": "3.32",
      "version": "3.32"
    },
    "NSSSMIME": {
      "minVersion": "3.32",
      "version": "3.32"
    }
  },
  "userJS": {
    "exists": false
  },
  "graphics": {
    "numTotalWindows": 11,
    "numAcceleratedWindows": 0,
    "windowLayerManagerType": "Basic",
    "windowLayerManagerRemote": true,
    "currentAudioBackend": "pulse",
    "numAcceleratedWindowsMessage": [
      "blockedGfxCard"
    ],
    "adapterDescription": "VMware, Inc. -- Gallium 0.4 on llvmpipe (LLVM 4.0, 256 bits)",
    "adapterVendorID": "VMware, Inc.",
    "adapterDeviceID": "Gallium 0.4 on llvmpipe (LLVM 4.0, 256 bits)",
    "adapterRAM": "",
    "adapterDrivers": "",
    "driverVersion": "3.0 Mesa 17.1.5",
    "driverDate": "",
    "webgl1Renderer": "VMware, Inc. -- Gallium 0.4 on llvmpipe (LLVM 4.0, 256 bits)",
    "webgl1Version": "3.0 Mesa 17.1.5",
    "webgl1DriverExtensions": "GL_ARB_multisample GL_EXT_abgr GL_EXT_bgra GL_EXT_blend_color GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_copy_texture GL_EXT_polygon_offset GL_EXT_subtexture GL_EXT_texture_object GL_EXT_vertex_array GL_EXT_compiled_vertex_array GL_EXT_texture GL_EXT_texture3D GL_IBM_rasterpos_clip GL_ARB_point_parameters GL_EXT_draw_range_elements GL_EXT_packed_pixels GL_EXT_point_parameters GL_EXT_rescale_normal GL_EXT_separate_specular_color GL_EXT_texture_edge_clamp GL_SGIS_generate_mipmap GL_SGIS_texture_border_clamp GL_SGIS_texture_edge_clamp GL_SGIS_texture_lod GL_ARB_framebuffer_sRGB GL_ARB_multitexture GL_EXT_framebuffer_sRGB GL_IBM_multimode_draw_arrays GL_IBM_texture_mirrored_repeat GL_ARB_texture_cube_map GL_ARB_texture_env_add GL_ARB_transpose_matrix GL_EXT_blend_func_separate GL_EXT_fog_coord GL_EXT_multi_draw_arrays GL_EXT_secondary_color GL_EXT_texture_env_add GL_EXT_texture_lod_bias GL_INGR_blend_func_separate GL_NV_blend_square GL_NV_light_max_exponent GL_NV_texgen_reflection GL_NV_texture_env_combine4 GL_S3_s3tc GL_SUN_multi_draw_arrays GL_ARB_texture_border_clamp GL_ARB_texture_compression GL_EXT_framebuffer_object GL_EXT_texture_compression_s3tc GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_MESA_window_pos GL_NV_packed_depth_stencil GL_NV_texture_rectangle GL_ARB_depth_texture GL_ARB_occlusion_query GL_ARB_shadow GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_mirrored_repeat GL_ARB_window_pos GL_ATI_fragment_shader GL_EXT_stencil_two_side GL_EXT_texture_cube_map GL_NV_depth_clamp GL_NV_fog_distance GL_APPLE_packed_pixels GL_APPLE_vertex_array_object GL_ARB_draw_buffers GL_ARB_fragment_program GL_ARB_fragment_shader GL_ARB_shader_objects GL_ARB_vertex_program GL_ARB_vertex_shader GL_ATI_draw_buffers GL_ATI_texture_env_combine3 GL_ATI_texture_float GL_EXT_shadow_funcs GL_EXT_stencil_wrap GL_MESA_pack_invert GL_MESA_ycbcr_texture GL_NV_primitive_restart GL_ARB_depth_clamp GL_ARB_fragment_program_shadow GL_ARB_half_float_pixel GL_ARB_occlusion_query2 GL_ARB_point_sprite GL_ARB_shading_language_100 GL_ARB_sync GL_ARB_texture_non_power_of_two GL_ARB_vertex_buffer_object GL_ATI_blend_equation_separate GL_EXT_blend_equation_separate GL_OES_read_format GL_ARB_color_buffer_float GL_ARB_pixel_buffer_object GL_ARB_texture_compression_rgtc GL_ARB_texture_float GL_ARB_texture_rectangle GL_ATI_texture_compression_3dc GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_mirror_clamp GL_EXT_texture_rectangle GL_EXT_texture_sRGB GL_EXT_texture_shared_exponent GL_ARB_framebuffer_object GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_packed_depth_stencil GL_ARB_vertex_array_object GL_ATI_separate_stencil GL_ATI_texture_mirror_once GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_gpu_program_parameters GL_EXT_texture_array GL_EXT_texture_compression_latc GL_EXT_texture_integer GL_EXT_texture_sRGB_decode GL_EXT_timer_query GL_OES_EGL_image GL_ARB_copy_buffer GL_ARB_depth_buffer_float GL_ARB_draw_instanced GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_map_buffer_range GL_ARB_texture_rg GL_ARB_texture_swizzle GL_ARB_vertex_array_bgra GL_EXT_texture_swizzle GL_EXT_vertex_array_bgra GL_NV_conditional_render GL_AMD_conservative_depth GL_AMD_draw_buffers_blend GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_ARB_ES2_compatibility GL_ARB_blend_func_extended GL_ARB_debug_output GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_explicit_attrib_location GL_ARB_fragment_coord_conventions GL_ARB_provoking_vertex GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_shader_stencil_export GL_ARB_shader_texture_lod GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_multisample GL_ARB_texture_rgb10_a2ui GL_ARB_uniform_buffer_object GL_ARB_vertex_type_2_10_10_10_rev GL_EXT_provoking_vertex GL_EXT_texture_snorm GL_MESA_texture_signed_rgba GL_ARB_get_program_binary GL_ARB_robustness GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_base_instance GL_ARB_compressed_texture_pixel_storage GL_ARB_conservative_depth GL_ARB_internalformat_query GL_ARB_map_buffer_alignment GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_texture_storage GL_ARB_transform_feedback_instanced GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_transform_feedback GL_AMD_shader_trinary_minmax GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_clear_buffer_object GL_ARB_copy_image GL_ARB_explicit_uniform_location GL_ARB_invalidate_subdata GL_ARB_program_interface_query GL_ARB_stencil_texturing GL_ARB_texture_query_levels GL_ARB_texture_storage_multisample GL_ARB_texture_view GL_ARB_vertex_attrib_binding GL_KHR_debug GL_ARB_buffer_storage GL_ARB_clear_texture GL_ARB_internalformat_query2 GL_ARB_multi_bind GL_ARB_seamless_cubemap_per_texture GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_stencil8 GL_ARB_vertex_type_10f_11f_11f_rev GL_EXT_shader_integer_mix GL_ARB_clip_control GL_ARB_conditional_render_inverted GL_ARB_cull_distance GL_ARB_get_texture_sub_image GL_EXT_polygon_offset_clamp GL_KHR_context_flush_control GL_MESA_shader_integer_functions ",
    "webgl1Extensions": "ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_frag_depth EXT_sRGB EXT_shader_texture_lod EXT_disjoint_timer_query MOZ_debug OES_element_index_uint OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_etc WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context MOZ_WEBGL_lose_context MOZ_WEBGL_depth_texture",
    "webgl1WSIInfo": "GLX 1.4\nGLX_VENDOR(client): Mesa Project and SGI\nGLX_VENDOR(server): SGI\nExtensions: GLX_ARB_create_context GLX_ARB_create_context_profile GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_import_context GLX_EXT_visual_info GLX_EXT_visual_rating GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_MESA_copy_sub_buffer GLX_MESA_multithread_makecurrent GLX_MESA_query_renderer GLX_OML_swap_method GLX_SGI_make_current_read GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_EXT_texture_from_pixmap ",
    "webgl2Renderer": "VMware, Inc. -- Gallium 0.4 on llvmpipe (LLVM 4.0, 256 bits)",
    "webgl2Version": "3.3 (Core Profile) Mesa 17.1.5",
    "webgl2DriverExtensions": "GL_AMD_conservative_depth GL_AMD_draw_buffers_blend GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_AMD_shader_trinary_minmax GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_ES2_compatibility GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_compressed_texture_pixel_storage GL_ARB_conditional_render_inverted GL_ARB_conservative_depth GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_draw_indirect GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_shader GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_occlusion_query2 GL_ARB_pixel_buffer_object GL_ARB_point_sprite GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_robustness GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_objects GL_ARB_shader_stencil_export GL_ARB_shader_subroutine GL_ARB_shader_texture_lod GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map_array GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ATI_blend_equation_separate GL_ATI_texture_float GL_ATI_texture_mirror_once GL_EXT_abgr GL_EXT_blend_equation_separate GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_sRGB GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_polygon_offset_clamp GL_EXT_provoking_vertex GL_EXT_shader_integer_mix GL_EXT_texture_array GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_integer GL_EXT_texture_mirror_clamp GL_EXT_texture_sRGB GL_EXT_texture_sRGB_decode GL_EXT_texture_shared_exponent GL_EXT_texture_snorm GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback GL_EXT_vertex_array_bgra GL_IBM_multimode_draw_arrays GL_KHR_context_flush_control GL_KHR_debug GL_MESA_pack_invert GL_MESA_shader_integer_functions GL_MESA_texture_signed_rgba GL_MESA_ycbcr_texture GL_NV_conditional_render GL_NV_depth_clamp GL_NV_packed_depth_stencil GL_OES_EGL_image GL_S3_s3tc",
    "webgl2Extensions": "EXT_color_buffer_float EXT_disjoint_timer_query MOZ_debug OES_texture_float_linear WEBGL_compressed_texture_etc WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context MOZ_WEBGL_lose_context",
    "webgl2WSIInfo": "GLX 1.4\nGLX_VENDOR(client): Mesa Project and SGI\nGLX_VENDOR(server): SGI\nExtensions: GLX_ARB_create_context GLX_ARB_create_context_profile GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_import_context GLX_EXT_visual_info GLX_EXT_visual_rating GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_MESA_copy_sub_buffer GLX_MESA_multithread_makecurrent GLX_MESA_query_renderer GLX_OML_swap_method GLX_SGI_make_current_read GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_EXT_texture_from_pixmap ",
    "info": {
      "AzureCanvasBackend": "skia",
      "AzureFallbackCanvasBackend": "none",
      "AzureContentBackend": "skia",
      "AzureCanvasAccelerated": 0,
      "CairoUseXRender": 0
    },
    "featureLog": {
      "features": [
        {
          "name": "HW_COMPOSITING",
          "description": "Compositing",
          "status": "blocked",
          "log": [
            {
              "type": "default",
              "status": "blocked",
              "message": "Acceleration blocked by platform"
            }
          ]
        },
        {
          "name": "OPENGL_COMPOSITING",
          "description": "OpenGL Compositing",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "unavailable",
              "message": "Hardware compositing is disabled"
            }
          ]
        },
        {
          "name": "WEBRENDER",
          "description": "WebRender",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "opt-in",
              "message": "WebRender is an opt-in feature"
            },
            {
              "type": "runtime",
              "status": "unavailable",
              "message": "Build doesn't include WebRender"
            }
          ]
        }
      ],
      "fallbacks": []
    },
    "crashGuards": []
  },
  "experiments": [],
  "extensions": [
    {
      "name": "About sessionstore",
      "version": "0.34",
      "isActive": true,
      "id": "aboutsessionstore@dt"
    },
    {
      "name": "AdBlock",
      "version": "3.4.1",
      "isActive": true,
      "id": "jid1-NIfFY2CA8fy1tg@jetpack"
    },
    {
      "name": "Add-on Compatibility Reporter",
      "version": "2.2.4",
      "isActive": true,
      "id": "compatibility@addons.mozilla.org"
    },
    {
      "name": "Add-ons Manager - Version Number",
      "version": "1.5",
      "isActive": true,
      "id": "AMVersionNumber@Aris_CTRdev"
    },
    {
      "name": "Advanced History",
      "version": "1.3",
      "isActive": true,
      "id": "{9f64c098-f00c-49b0-922f-9d43a9c2278e}"
    },
    {
      "name": "Archive URL",
      "version": "1.1",
      "isActive": true,
      "id": "@save-url-to-wayback-machine-firefox-addon"
    },
    {
      "name": "Auto Unload Tab",
      "version": "0.9.1.5",
      "isActive": true,
      "id": "{d3c46ca0-999d-11da-a72b-0800200c9a66}"
    },
    {
      "name": "Binghodie",
      "version": "1.2.0",
      "isActive": true,
      "id": "{645b9240-4394-4f0d-a0be-c2c21ea5f08d}"
    },
    {
      "name": "Bookmarks Organizer",
      "version": "1.3.0",
      "isActive": true,
      "id": "bookmarksorganizer@agenedia.com"
    },
    {
      "name": "Clear Tabs",
      "version": "1.0.5.1",
      "isActive": true,
      "id": "{442cfece-ae4e-49da-8ec2-c3f984608d60}"
    },
    {
      "name": "ClearUrls",
      "version": "0.4",
      "isActive": true,
      "id": "{74145f27-f039-47ce-a470-a662b129930a}"
    },
    {
      "name": "Configuration Mania",
      "version": "25.5.0",
      "isActive": true,
      "id": "{c4d362ec-1cff-4ca0-9031-99a8fad7995a}"
    },
    {
      "name": "Containers Experiment",
      "version": "2.4.1",
      "isActive": true,
      "id": "@testpilot-containers"
    },
    {
      "name": "Context Plus",
      "version": "0.3.3",
      "isActive": true,
      "id": "{bb682c45-3136-4213-bf29-5f5833080bf4}"
    },
    {
      "name": "Copy Page Title (WebExtension)",
      "version": "1.5",
      "isActive": true,
      "id": "cpt@mozilla.org"
    },
    {
      "name": "Custom UserAgent String",
      "version": "0.1.6",
      "isActive": true,
      "id": "{861a3982-bb3b-49c6-bc17-4f50de104da1}"
    },
    {
      "name": "Disconnect",
      "version": "5.18.21",
      "isActive": true,
      "id": "2.0@disconnect.me"
    },
    {
      "name": "Don't touch my tabs! (rel=noopener)",
      "version": "1.0",
      "isActive": true,
      "id": "{6b938c0c-fc53-4f27-805f-619778631082}"
    },
    {
      "name": "Dorando keyconfig",
      "version": "2016.2",
      "isActive": true,
      "id": "keyconfig@mozilla.dorando.at"
    },
    {
      "name": "Extension source viewer",
      "version": "1.6",
      "isActive": true,
      "id": "crxviewer-firefox@robwu.nl"
    },
    {
      "name": "FireTitle",
      "version": "0.5.5-1.1-signed.1-signed.1-signed",
      "isActive": true,
      "id": "{f4b962b4-ab75-41bf-8da7-a0435258a27c}"
    },
    {
      "name": "Font Inspector",
      "version": "0.1.1",
      "isActive": true,
      "id": "{39339c9f-e978-41ec-ba7e-18576c3b1253}"
    },
    {
      "name": "FoxyAbout",
      "version": "0.1.1",
      "isActive": true,
      "id": "jid1-d3SwJwS6q9JpMq@jetpack"
    },
    {
      "name": "Free Art Tab",
      "version": "1.0.11",
      "isActive": true,
      "id": "{d5b60e1d-bc94-4d42-8baf-e24c4505ed48}"
    },
    {
      "name": "Google search link fix",
      "version": "1.6.5",
      "isActive": true,
      "id": "jid0-XWJxt5VvCXkKzQK99PhZqAn7Xbg@jetpack"
    },
    {
      "name": "InlineDisposition (WebExtensions)",
      "version": "1.4",
      "isActive": true,
      "id": "{b134ae49-6ada-41e0-9141-9f8d00373d93}"
    },
    {
      "name": "LastPass: Free Password Manager",
      "version": "4.1.54",
      "isActive": true,
      "id": "support@lastpass.com"
    },
    {
      "name": "Link Cleaner",
      "version": "1.5",
      "isActive": true,
      "id": "{6d85dea2-0fb4-4de3-9f8c-264bce9a2296}"
    },
    {
      "name": "MuteTab",
      "version": "0.0.2",
      "isActive": true,
      "id": "mutetab@mozilla.org"
    },
    {
      "name": "Print Edit WE",
      "version": "19.0",
      "isActive": true,
      "id": "printedit-we@DW-dev"
    },
    {
      "name": "Privacy Badger",
      "version": "2017.7.24",
      "isActive": true,
      "id": "jid1-MnnxcxisBPnSXQ@jetpack"
    },
    {
      "name": "Rename Tab Title",
      "version": "1.0",
      "isActive": true,
      "id": "renametabtitle@mozilla.org"
    },
    {
      "name": "Replaycer",
      "version": "1.1",
      "isActive": true,
      "id": "{4da3a448-ddc6-424a-a5c8-5af27c3516ac}"
    },
    {
      "name": "RevEye Reverse Image Search",
      "version": "1.4.5",
      "isActive": true,
      "id": "{1526fba1-ac33-4dfc-99d8-163e6129f7b9}"
    },
    {
      "name": "Rhombus",
      "version": "1.3",
      "isActive": true,
      "id": "{c8a68c13-2f68-4325-b963-9f08afbde63e}"
    },
    {
      "name": "Save Page WE",
      "version": "7.0",
      "isActive": true,
      "id": "savepage-we@DW-dev"
    },
    {
      "name": "Search Panel in Context Menu",
      "version": "2.0.26",
      "isActive": true,
      "id": "BestSearch@LukaszPolowczyk"
    },
    {
      "name": "Send to VLC (VideoLAN) media player",
      "version": "0.1.7",
      "isActive": true,
      "id": "{3e0ac434-26e0-4c03-b757-3078486800c3}"
    },
    {
      "name": "Session Manager",
      "version": "0.8.1.13",
      "isActive": true,
      "id": "{1280606b-2510-4fe0-97ef-9b5a22eafe30}"
    },
    {
      "name": "Slim Add-ons Manager",
      "version": "14.1",
      "isActive": true,
      "id": "slimaddonmanager@opendfki.de"
    },
    {
      "name": "Smart Tab Mute",
      "version": "1.0.1.1",
      "isActive": true,
      "id": "smart-tab-mute@die-antwort.eu"
    },
    {
      "name": "Sort tabs",
      "version": "1.2",
      "isActive": true,
      "id": "{418e6a02-1367-4204-8dc8-0c58eb12d837}"
    },
    {
      "name": "Speed Tweaks (SpeedyFox)",
      "version": "0.2.2",
      "isActive": true,
      "id": "jid1-wZqm19rJzRkZUA@jetpack"
    },
    {
      "name": "Stylish",
      "version": "2.0.7",
      "isActive": true,
      "id": "{46551EC9-40F0-4e47-8E18-8E5CF550CFB8}"
    },
    {
      "name": "Tab Center Redux",
      "version": "0.4.0",
      "isActive": true,
      "id": "{0ad88674-2b41-4cfb-99e3-e206c74a0076}"
    },
    {
      "name": "Tab Groups",
      "version": "2.1.4",
      "isActive": true,
      "id": "tabgroups@quicksaver"
    },
    {
      "name": "Tab Reloader (page reloader)",
      "version": "0.1.9",
      "isActive": true,
      "id": "jid0-bnmfwWw2w2w4e4edvcdDbnMhdVg@jetpack"
    },
    {
      "name": "Tab Scope",
      "version": "1.6.3",
      "isActive": true,
      "id": "tabscope@xuldev.org"
    },
    {
      "name": "Tab Tally",
      "version": "1.2.0",
      "isActive": true,
      "id": "@tab-tally"
    },
    {
      "name": "Vertical Tabs Reloaded",
      "version": "0.8.4",
      "isActive": true,
      "id": "verticaltabsreloaded@go-dev.de"
    },
    {
      "name": "You're Not So Special",
      "version": "1.1",
      "isActive": true,
      "id": "you-re-not-so-special@betacantrips.com"
    },
    {
      "name": "404 : File is Not Found ? Now it will be!",
      "version": "1.4.1-signed.1-signed",
      "isActive": false,
      "id": "waybackbutton@lazar.kovacevic"
    },
    {
      "name": "404 Bookmarks",
      "version": "1.1",
      "isActive": false,
      "id": "{5f8d31ba-47fb-4b70-bf8d-d2113f6da22f}"
    },
    {
      "name": "Adblock Plus",
      "version": "2.9.1",
      "isActive": false,
      "id": "{d10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d}"
    },
    {
      "name": "Always Preview (WebExtension)",
      "version": "1.8.5",
      "isActive": false,
      "id": "{b434be68-4cab-41e0-9141-9f8d00373d93}"
    },
    {
      "name": "ArchiveThis",
      "version": "1.1.0",
      "isActive": false,
      "id": "@archivethis"
    },
    {
      "name": "Art Project",
      "version": "0.1.5",
      "isActive": false,
      "id": "jid1-2owcJCGUIo2yBA@jetpack"
    },
    {
      "name": "Auto mute",
      "version": "1.0",
      "isActive": false,
      "id": "autoMute@waldemar.b"
    },
    {
      "name": "AutoHiDPI",
      "version": "3.0.1",
      "isActive": false,
      "id": "jid1-ylDsmQrKspNYVg@jetpack"
    },
    {
      "name": "Awesome Screenshot - Capture, Annotate & More",
      "version": "3.0.21",
      "isActive": false,
      "id": "jid0-GXjLLfbCoAx0LcltEdFrEkQdQPI@jetpack"
    },
    {
      "name": "Best Theme like Google Material Design Paper Theme",
      "version": "1.1.14",
      "isActive": false,
      "id": "BestTheme@LukaszPolowczyk"
    },
    {
      "name": "Beyond Australis",
      "version": "1.4.8",
      "isActive": false,
      "id": "thefoxonlybetter@quicksaver"
    },
    {
      "name": "Bookmark Shortcut Keys",
      "version": "0.76.1-signed.1-signed",
      "isActive": false,
      "id": "shortcut-keys@jbalogh.me"
    },
    {
      "name": "Bookmarklets context menu",
      "version": "1.1.3",
      "isActive": false,
      "id": "{8d2a1b8a-2ff6-4bde-aae4-4a9dd54b7e66}"
    },
    {
      "name": "Cache Disabler",
      "version": "1.2.0",
      "isActive": false,
      "id": "@cachedisabler"
    },
    {
      "name": "CacheViewer",
      "version": "0.8.6.3.1-let-fixed",
      "isActive": false,
      "id": "{71328583-3CA7-4809-B4BA-570A85818FBB}"
    },
    {
      "name": "ChatZilla",
      "version": "0.9.93",
      "isActive": false,
      "id": "{59c81df5-4b7a-477b-912d-4e0fdf64e5f2}"
    },
    {
      "name": "Chrome Store Foxified",
      "version": "2.4",
      "isActive": false,
      "id": "Chrome-Store-Foxified@jetpack"
    },
    {
      "name": "Cleanest Addon Manager",
      "version": "7.1",
      "isActive": false,
      "id": "cam@sdrocking.com"
    },
    {
      "name": "Click to Play per-element",
      "version": "0.3.4",
      "isActive": false,
      "id": "ClickToPlayPerElement@uaSad.addons.mozilla.org"
    },
    {
      "name": "Contrast Fix",
      "version": "1.0.0",
      "isActive": false,
      "id": "{80f6f2e4-eda1-417f-bf54-9645e1e20f5d}"
    },
    {
      "name": "Copy Link Title (WebExtensions)",
      "version": "1.4.2",
      "isActive": false,
      "id": "clt@mozilla.org"
    },
    {
      "name": "Custom Tab Width",
      "version": "1.0.1.1-signed.1-signed",
      "isActive": false,
      "id": "tab-width@design-noir.de"
    },
    {
      "name": "Digital Eye Strain Prevention Tool",
      "version": "1.1.1",
      "isActive": false,
      "id": "{76f406ef-a84c-4c71-80e2-0e3e78b90265}"
    },
    {
      "name": "Diigo Toolbar",
      "version": "5.1.0.38.1-signed.1-signed",
      "isActive": false,
      "id": "{fc2b8f80-d9a5-4f51-8076-7c7ce3c67ee3}"
    },
    {
      "name": "Display #Anchors",
      "version": "1.3",
      "isActive": false,
      "id": "display-anchors@robwu.nl"
    },
    {
      "name": "Easy Youtube Video Downloader Express",
      "version": "9.16",
      "isActive": false,
      "id": "{b9acf540-acba-11e1-8ccb-001fd0e08bd4}"
    },
    {
      "name": "EnGrip Extension",
      "version": "0.11.1",
      "isActive": false,
      "id": "tracker@engrip.com"
    },
    {
      "name": "eyeguard",
      "version": "2.3.5.1-signed",
      "isActive": false,
      "id": "jid1-4kGswXOBHEsvhQ@jetpack"
    },
    {
      "name": "File and Folder Shortcuts",
      "version": "1.4.1-signed.1-signed",
      "isActive": false,
      "id": "{3bbdd952-cf6f-44a7-9d23-354a8792b598}"
    },
    {
      "name": "Firebug",
      "version": "2.0.19",
      "isActive": false,
      "id": "firebug@software.joehewitt.com"
    },
    {
      "name": "Firefogg",
      "version": "356",
      "isActive": false,
      "id": "firefogg@firefogg.org"
    },
    {
      "name": "Firefox Pioneer",
      "version": "1.0",
      "isActive": false,
      "id": "pioneer-opt-in@mozilla.org"
    },
    {
      "name": "Force Content-Type",
      "version": "1.2.2.1-signed.1-signed",
      "isActive": false,
      "id": "forcecontenttype@alberto.silva"
    },
    {
      "name": "GitHub Extension Installer",
      "version": "1.7.0",
      "isActive": false,
      "id": "{86054B0A-BD85-42F9-8E58-8794EC6F6EA1}"
    },
    {
      "name": "GitHub Streaks",
      "version": "1.5",
      "isActive": false,
      "id": "github-streaks@mozilla.org"
    },
    {
      "name": "Google Keep Notes",
      "version": "50.1",
      "isActive": false,
      "id": "{a94d60a0-8408-4c53-8eec-cb349eb958b8}"
    },
    {
      "name": "Google PDF Viewer",
      "version": "1.3.7.4",
      "isActive": false,
      "id": "{b734ba68-4cab-41e0-9141-9f8d00373d93}"
    },
    {
      "name": "Grammarly for Firefox",
      "version": "8.775.1034",
      "isActive": false,
      "id": "87677a2c52b84ad3a151a4a72f5bd3c4@jetpack"
    },
    {
      "name": "Hide Tab Bar With One Tab",
      "version": "1.5.1-signed.1-signed",
      "isActive": false,
      "id": "{e5bbc237-c99b-4ced-a061-0be27703295f}"
    },
    {
      "name": "Horizontal Scroll",
      "version": "0.1.1",
      "isActive": false,
      "id": "@horizontal-scroll"
    },
    {
      "name": "Hotkey Tabs",
      "version": "1.7.3",
      "isActive": false,
      "id": "@hotkeytabs_release"
    },
    {
      "name": "Ideal Size",
      "version": "1.0.1",
      "isActive": false,
      "id": "idealsize@mozilla.kewis.ch"
    },
    {
      "name": "Image Search for Tineye",
      "version": "1.0.0",
      "isActive": false,
      "id": "image_search_for_tiney@image_search_for_tiney.org"
    },
    {
      "name": "Inline Audio Player",
      "version": "0.1.1.1-signed.1-signed",
      "isActive": false,
      "id": "jid1-FUBXJBBKIeigHQ@jetpack"
    },
    {
      "name": "Integrated Authentication for Firefox",
      "version": "3.0.1.1-signed.1-signed",
      "isActive": false,
      "id": "extension@firefox-ntlmauth.googlecode.com"
    },
    {
      "name": "Local Music Player",
      "version": "0.11.0.1-signed.1-signed",
      "isActive": false,
      "id": "jid1-nCt6CO1kXSzfAg@jetpack"
    },
    {
      "name": "Long Click",
      "version": "1.2.1",
      "isActive": false,
      "id": "longclick10@epistemex.com"
    },
    {
      "name": "Min Vid",
      "version": "0.3.9",
      "isActive": false,
      "id": "@min-vid"
    },
    {
      "name": "Mozilla Archive Format",
      "version": "4.1.3",
      "isActive": false,
      "id": "{7f57cf46-4467-4c2d-adfa-0cba7c507e54}"
    },
    {
      "name": "mSession Keeper",
      "version": "0.7.51",
      "isActive": false,
      "id": "multi-session-keeper@pmorzonek"
    },
    {
      "name": "Multiple Tab Handler",
      "version": "0.8.2017061501",
      "isActive": false,
      "id": "multipletab@piro.sakura.ne.jp"
    },
    {
      "name": "MultiSelect Tabs",
      "version": "0.4.0",
      "isActive": false,
      "id": "multi-select-tabs@mikeconley.ca"
    },
    {
      "name": "Mute all inactive tabs",
      "version": "0.1.1",
      "isActive": false,
      "id": "mute-all-inactive-tabs@evilpie.tomschuster.name"
    },
    {
      "name": "Online Reload",
      "version": "1.0.1-signed.1-signed",
      "isActive": false,
      "id": "onlinereload@braniecki.net"
    },
    {
      "name": "Open in Browser",
      "version": "1.18",
      "isActive": false,
      "id": "openinbrowser@www.spasche.net"
    },
    {
      "name": "Open in Wayback Machine",
      "version": "2.0.2",
      "isActive": false,
      "id": "jid1-D3J3CDl5LFaX5A@jetpack"
    },
    {
      "name": "Open Link in Silent Tab",
      "version": "0.2.5.1-signed.1-signed",
      "isActive": false,
      "id": "{d4c46ca0-999d-11da-a72b-0800200c9a66}"
    },
    {
      "name": "Other Window",
      "version": "1.0",
      "isActive": false,
      "id": "{5df6e133-f35d-4c62-885a-56387df22f6b}"
    },
    {
      "name": "Page Screenshot",
      "version": "0.1.0",
      "isActive": false,
      "id": "{f52149fe-80cc-4d07-868d-c0e4a85453a0}"
    },
    {
      "name": "PDFescape Extension",
      "version": "0.17.1-signed.1-signed",
      "isActive": false,
      "id": "{2A1D5949-B519-4924-BF62-8522FE0D5274}"
    },
    {
      "name": "pdfit",
      "version": "1.15.1-signed.1-signed",
      "isActive": false,
      "id": "service@touchpdf.com"
    },
    {
      "name": "ProfileSwitcher",
      "version": "1.7.6.1",
      "isActive": false,
      "id": "{fa8476cf-a98c-4e08-99b4-65a69cb4b7d4}"
    },
    {
      "name": "Quick Locale Switcher",
      "version": "1.7.8.5.1-signed.1-signed",
      "isActive": false,
      "id": "{25A1388B-6B18-46c3-BEBA-A81915D0DE8F}"
    },
    {
      "name": "Re-search",
      "version": "1.1.8",
      "isActive": false,
      "id": "{d0064f8d-e9e6-4a60-a712-eab45d195a81}"
    },
    {
      "name": "Reload on Idle",
      "version": "1.6",
      "isActive": false,
      "id": "reload-on-idle@clear-code.com"
    },
    {
      "name": "Reload Plus",
      "version": "5.2.3",
      "isActive": false,
      "id": "reloadplus@blackwind"
    },
    {
      "name": "Resurrect Pages (isup edition)",
      "version": "3.0.0",
      "isActive": false,
      "id": "resurrect-pages@albirew.fr"
    },
    {
      "name": "Reverse Image Search",
      "version": "0.2.0",
      "isActive": false,
      "id": "jid0-iiiWfb12bgHj8iKloOou74fb6jh@jetpack"
    },
    {
      "name": "S3.Google Translator",
      "version": "5.32",
      "isActive": false,
      "id": "s3google@translator"
    },
    {
      "name": "Scrollbar Auto Show/Hide",
      "version": "2.6",
      "isActive": false,
      "id": "{d3d35bb5-999d-11da-a72b-0800200c9a66}"
    },
    {
      "name": "Sea Containers",
      "version": "0.5",
      "isActive": false,
      "id": "{d975a11d-08cd-4aea-b7c0-989209ad860f}"
    },
    {
      "name": "Search by Image for Google",
      "version": "1.2.0.1-signed.1-signed",
      "isActive": false,
      "id": "{ab4b5718-3998-4a2c-91ae-18a7c2db513e}"
    },
    {
      "name": "Simplified Tab Groups",
      "version": "0.5.1",
      "isActive": false,
      "id": "tabgroups@schub.io"
    },
    {
      "name": "snoozetabs",
      "version": "1.0.19",
      "isActive": false,
      "id": "snoozetabs@mozilla.com"
    },
    {
      "name": "Sound Control",
      "version": "1.2.0",
      "isActive": false,
      "id": "{67b2c222-9d4e-4747-ad68-39e8ba76ca3b}"
    },
    {
      "name": "SQLite Manager",
      "version": "0.8.3.1-signed.1-signed",
      "isActive": false,
      "id": "SQLiteManager@mrinalkant.blogspot.com"
    },
    {
      "name": "Stylish Sync",
      "version": "0.1.7.1-signed.1-signed",
      "isActive": false,
      "id": "{0e3fc079-afbb-4a00-87e5-9486062d0f9c}"
    },
    {
      "name": "Stylus",
      "version": "1.1.3.1",
      "isActive": false,
      "id": "{7a7a4a92-a2a0-41d1-9fd7-1e92480d612d}"
    },
    {
      "name": "Sum Tabs",
      "version": "1.0",
      "isActive": false,
      "id": "{7d2b5a89-40aa-4f7c-b8c8-8013a3be8abe}"
    },
    {
      "name": "Support Free Content",
      "version": "2.1.3",
      "isActive": false,
      "id": "supportfreecontent@mozilla.org"
    },
    {
      "name": "Tab Count",
      "version": "0.0.1",
      "isActive": false,
      "id": "{b1f2a5fa-600c-4a9a-bb5d-ec3849f1bbe7}"
    },
    {
      "name": "Tab Count Icon",
      "version": "0.6",
      "isActive": false,
      "id": "{8e671754-fc9e-41c0-bb7c-0f575c4127d6}"
    },
    {
      "name": "Tab Counter",
      "version": "0.1.0",
      "isActive": false,
      "id": "tab-counter@daawesomep.addons.mozilla.org"
    },
    {
      "name": "Tab Groups Helper",
      "version": "1.0.30",
      "isActive": false,
      "id": "tabgroupshelper@kevinallasso.org"
    },
    {
      "name": "Tab Memory Usage",
      "version": "0.2.7",
      "isActive": false,
      "id": "jid1-fRvgLzKONCsPew@jetpack"
    },
    {
      "name": "Tab Mover",
      "version": "4",
      "isActive": false,
      "id": "tab-mover@code.guido-berhoerster.org"
    },
    {
      "name": "Tab Suspender (memory saver)",
      "version": "0.1.8",
      "isActive": false,
      "id": "{e225ac78-5e83-484b-a16b-b6ed0924212f}"
    },
    {
      "name": "Tab Wrangler",
      "version": "0.11",
      "isActive": false,
      "id": "{f82cbc08-ccb2-4adb-b2ec-f72310650602}"
    },
    {
      "name": "tab-count-button",
      "version": "1.0",
      "isActive": false,
      "id": "{c78d9d72-ca51-4f41-83dc-4c7f6ea346b1}"
    },
    {
      "name": "tabs2txt",
      "version": "1.1",
      "isActive": false,
      "id": "{70f84a3b-6308-43c7-8373-25873cdab5cc}"
    },
    {
      "name": "Tampermonkey",
      "version": "4.4.5484",
      "isActive": false,
      "id": "firefox@tampermonkey.net"
    },
    {
      "name": "Test Pilot",
      "version": "2.0.2-dev-aa555d2",
      "isActive": false,
      "id": "@testpilot-addon"
    },
    {
      "name": "Text Contrast for Dark Themes",
      "version": "1.4.1",
      "isActive": false,
      "id": "jid1-nMVE2oP40qeQDQ@jetpack"
    },
    {
      "name": "Tile Tabs WE",
      "version": "6.3",
      "isActive": false,
      "id": "tiletabs-we@DW-dev"
    },
    {
      "name": "TinEye Reverse Image Search",
      "version": "1.3.2",
      "isActive": false,
      "id": "tineye@ideeinc.com"
    },
    {
      "name": "torelo",
      "version": "0.5",
      "isActive": false,
      "id": "{89b5159d-9840-4549-9fbe-72286591c4e1}"
    },
    {
      "name": "Tree Style Tab",
      "version": "0.19.2017061601",
      "isActive": false,
      "id": "treestyletab@piro.sakura.ne.jp"
    },
    {
      "name": "Tree Tabs",
      "version": "0.0.16",
      "isActive": false,
      "id": "TreeTabs@jagiello.it"
    },
    {
      "name": "uMatrix",
      "version": "1.0.1b5",
      "isActive": false,
      "id": "uMatrix@raymondhill.net"
    },
    {
      "name": "Unified Sidebar",
      "version": "0.1.2011111101.1-signed.1-signed",
      "isActive": false,
      "id": "unifiedsidebar@piro.sakura.ne.jp"
    },
    {
      "name": "URL in Titlebar",
      "version": "1.1.1",
      "isActive": false,
      "id": "urlit@sudoreboot.org"
    },
    {
      "name": "URLs List",
      "version": "0.2.0",
      "isActive": false,
      "id": "{88664789-f91e-40e1-adb9-e4e9a8c48867}"
    },
    {
      "name": "User-Agent Switcher",
      "version": "0.1.9",
      "isActive": false,
      "id": "{75afe46a-7a50-4c6b-b866-c43a1075b071}"
    },
    {
      "name": "Vertigo Tabs",
      "version": "0.4",
      "isActive": false,
      "id": "{81b88459-b274-4e81-936c-e3caa19d46e0}"
    },
    {
      "name": "Video DownloadHelper",
      "version": "6.3.1",
      "isActive": false,
      "id": "{b9db16a4-6edc-47ec-a1f4-b86292ed211d}"
    },
    {
      "name": "Violentmonkey",
      "version": "2.6.3",
      "isActive": false,
      "id": "{aecec67f-0d10-4fa7-b7c7-609a2db280cf}"
    },
    {
      "name": "Web Developer",
      "version": "1.2.13",
      "isActive": false,
      "id": "{c45c406e-ab73-11d8-be73-000a95be3b12}"
    },
    {
      "name": "Web Messenger for WhatsApp™",
      "version": "54.0",
      "isActive": false,
      "id": "{e1ab1333-aff6-42fe-84a5-55be2409491b}"
    },
    {
      "name": "Web Mirror Viewer",
      "version": "1.95.1-signed.1-signed",
      "isActive": false,
      "id": "{B2EA3FAB-912C-48a1-BABD-C5B00BB885BB}"
    },
    {
      "name": "Who Targets Me",
      "version": "0.1.11",
      "isActive": false,
      "id": "info@whotargets.me"
    },
    {
      "name": "Window Master",
      "version": "7.5",
      "isActive": false,
      "id": "monitormaster@pjs.nl"
    },
    {
      "name": "Xmarks",
      "version": "4.4.1",
      "isActive": false,
      "id": "foxmarks@kei.com"
    },
    {
      "name": "YourOnlineChoices Plugin",
      "version": "0.1.1.1-signed.1-signed",
      "isActive": false,
      "id": "yoc@edaa.eu"
    }
  ],
  "features": [
    {
      "name": "Application Update Service Helper",
      "version": "2.0",
      "id": "aushelper@mozilla.org"
    },
    {
      "name": "Click-to-Play staged rollout",
      "version": "1.2",
      "id": "clicktoplay-rollout@mozilla.org"
    },
    {
      "name": "Firefox Screenshots",
      "version": "10.10.0",
      "id": "screenshots@mozilla.org"
    },
    {
      "name": "Follow-on Search Telemetry",
      "version": "0.9.1",
      "id": "followonsearch@mozilla.com"
    },
    {
      "name": "Multi-process staged rollout",
      "version": "2.0",
      "id": "e10srollout@mozilla.org"
    },
    {
      "name": "Pocket",
      "version": "1.0.5",
      "id": "firefox@getpocket.com"
    },
    {
      "name": "Shield Recipe Client",
      "version": "55.1",
      "id": "shield-recipe-client@mozilla.org"
    },
    {
      "name": "Web Compat",
      "version": "1.1",
      "id": "webcompat@mozilla.org"
    }
  ]
}
```

**Comment by grahamperrin** [comment]
Duplicate of #14. (Sorry! Whilst reporting 14, I completely forgot that I had already reported 13.)

**Comment by captnfab** [comment]
Wow, I hadn't seen this issue, I was in holiday :P
